logging/cluster-logging-release-notes.adoc

```diff
@@ -8,235 +8,14 @@ toc::[]
 [id="openshift-logging-5-0-about-this-release"]
 == About this release
 
-(link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]) is now available. New features, changes, and known issues that pertain to {ProductName} 5.0 are included in this topic.
+The following advisories are available for {ProductName} 5.0:
+
+* link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]
```

Comment:
@vikram-redhat  I might want to move this into modules/cluster-logging-release-notes-5.0.0.adoc. What do you think?